### PR TITLE
OPSEXP-586 use shared travis config

### PR DESCRIPTION
**Pull request by mteodori**
replaced custom tool install lines with shared configs

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-language: python
+slanguage: python
```
Comment (gavincornwell) [inline .travis.yml:1]: Typo?

File: .travis.yml [hunk 2]
```diff
@@ -20,6 +20,8 @@ stages:
 
 import:
   - source: Alfresco/alfresco-process-tools:.travis.kubernetes_install.yml@master
+  - source: Alfresco/alfresco-process-tools:.travis.helm.yml@master
```
Comment (gavincornwell) [inline .travis.yml:23]: Doing this breaks our build, see https://travis-ci.com/github/Alfresco/acs-deployment/builds/191249066 so it was omitted previously purposely. 

The publish task runs too early. I raised a support ticket with Travis previously, there is a way to fix the order but we have also customised the publish process slightly so can't use the generic one.

If we can decide on a common way of publishing charts we might be able to fix it.

**Review by gavincornwell — CHANGES_REQUESTED** [review]
The kubepug related changes look good but importing the common Helm script breaks the build :-(

File: .travis.yml (hunk 1 above)
Comment (mteodori) [reply .travis.yml:1]: definitely thanks

File: .travis.yml (hunk 2 above)
Comment (mteodori) [reply .travis.yml:23]: got it wrong sorry, it should have been just install

**Review by gavincornwell — APPROVED** [review]
LGTM

**Pull request merged**